docs/modules/datasources/pages/distributed.adoc

```diff
@@ -0,0 +1,31 @@
+= OpenNMS Kafka Datasource
+:imagesdir: ../assets/images
+
+== Overview
+
+The OpenNMS Kafka Datasource is used when OCE is deployed in a *distributed* installation. It runs in the same JVM as OCE and acts upon new *Alarms* and *Nodes* as they are consumed from Kafka topics.
```

Comment:
should we mention `data` instead of mentioning the discrete forms of data we consume(alarms/nodes)? Since this list actually includes alarm feedback as well as topology links?